src/frontend/js/components/Dashboard/index.spec.tsx

```diff
@@ -40,16 +44,24 @@ describe('<Dashboard />', () => {
     return (
       <IntlProvider locale="en">
         <QueryClientProvider client={client}>
-          <BaseSessionProvider>
-            <Dashboard />
-          </BaseSessionProvider>
+          <JoanieSessionProvider>
+            <DashboardTest initialRoute={DashboardPaths.COURSES} />
+          </JoanieSessionProvider>
         </QueryClientProvider>
       </IntlProvider>
     );
   };
 
   beforeEach(() => {
     jest.resetAllMocks();
+    fetchMock.get('https://joanie.endpoint/api/orders/', []);
+    fetchMock.get('https://joanie.endpoint/api/credit-cards/', []);
+    fetchMock.get('https://joanie.endpoint/api/addresses/', []);
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+    fetchMock.restore();
```

Comment:
Why do you need to do this ? From my point of view, `beforeEach` is already in charge to clean those things.

Reply:
I can suppress `jest.clearAllMock();` but I need to keep `fetchMock.restore();` to be able de define mocks before each tests.